templates/firefox-ios.xliff

```diff
@@ -1327,8 +1338,12 @@
         <source>Paste &amp; Go</source>
         <note>Paste the URL into the location bar and visit</note>
       </trans-unit>
+      <trans-unit id="PhotonMenu.cancel">
+        <source>Cancel</source>
+        <note>Button for closing the menu action sheet</note>
+      </trans-unit>
       <trans-unit id="Please allow Firefox to access your device’s camera in ‘Settings’ -&gt; ‘Privacy’ -&gt; ‘Camera’.">
-        <source>Please allow Firefox to access your device’s camera in ’Settings’ -&gt; ’Privacy’ -&gt; ’Camera’.</source>
+        <source>Please allow Firefox to access your device’s camera in ‘Settings’ -&gt; ‘Privacy’ -&gt; ‘Camera’.</source>
```

Comment:
Look like I didn't change this string correctly in my script. I'm personally OK with losing this one translation, the alternative is to fix the script, re-update the repo, and re-run the export